### PR TITLE
Paas 1681

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-1681

Short description:

### CODE REVIEW

**Review by laurentfufu — CHANGES_REQUESTED** [review]
Shouldn't `applicationcontext-purge-jobs.xml` be enabled on processing during an install?

I'm also thinking about backup and restore:
* If the file is present (and normally activated because it is saved on the processing) in the backup, it should be deactivated on the browsings
* If the file is not present because of a "old" backup, we can probably forgot about browsing nodes, but it should be downloaded and activated on the processing

**Comment by adoux** [comment]
> Shouldn't `applicationcontext-purge-jobs.xml` be enabled on processing during an install?
> 
> I'm also thinking about backup and restore:
> 
> * If the file is present (and normally activated because it is saved on the processing) in the backup, it should be deactivated on the browsings
> * If the file is not present because of a "old" backup, we can probably forgot about browsing nodes, but it should be downloaded and activated on the processing

Changes made

File: packages/common/restore.yml [hunk 1]
```diff
@@ -223,6 +223,24 @@ actions:
...
             type: error
             message: "An error occurred while restoring jahia data. ${response.errOut}"
+    # Content History purge job should only be enabled on the processing node
+    - cmd [cp]: |-
+        file=/opt/tomcat/conf/digital-factory-config/jahia/applicationcontext-purge-jobs.xml
+        if [ -f $file ]; then
+          mv $file ${file}-disabled
+        else
+          url=${globals.repoRootUrl}/packages/jahia/migrations/v20_assets/applicationcontext-purge-jobs.xml-disabled
+          curl -fSsLo ${file}-disabled $url || exit 1
```
Comment (laurentfufu) [inline packages/common/restore.yml:233]: i'm not sure it will be useful to download the file on browsing

File: packages/common/restore.yml [hunk 2]
```diff
@@ -223,6 +223,24 @@ actions:
...
+        fi
+      user: tomcat
+    - cmd [proc]: |-
+        file=/opt/tomcat/conf/digital-factory-config/jahia/applicationcontext-purge-jobs.xml
+        if ! [ -f $file ]; then
+          url=${globals.repoRootUrl}/packages/jahia/migrations/v20_assets/applicationcontext-purge-jobs.xml-disabled
```
Comment (laurentfufu) [inline packages/common/restore.yml:239]: I guess the file should be promoted to a real asset and should be available in `${globals.repoRootUrl}/assets/jahia`

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: packages/common/restore.yml (hunk 1 above)
Comment (adoux) [reply packages/common/restore.yml]: I removed this

File: packages/common/restore.yml (hunk 2 above)
Comment (adoux) [reply packages/common/restore.yml]: Good call, done

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**